pkg/config/engine/containerd/config.go

```diff
@@ -111,6 +104,13 @@ func (c Config) DefaultRuntime() string {
 	return ""
 }
 
+// EnableCDI sets the enable_cdi field in the Containerd config to true.
+func (c *Config) EnableCDI() {
+	config := *c.Tree
+	config.SetPath([]string{"plugins", c.CRIRuntimePluginName, "enable_cdi"}, true)
```

Comment:
nitpick: to add an explicit log for this enablement. something like 
`c.Logger.Infof("enabled CDI in %v ", c.CRIRuntimePluginName)`

Reply:
If we want to add a debug or log statement, it should be at the call site.